### PR TITLE
Add Release Notes for October Management Library Releases

### PR DESCRIPTION
**Pull request by nickzhums**
This is a draft PR to add release notes for all management libraries.
Previous PR is undone since not all packages have been released.

Will merge this PR only if the packages are all released. Thanks!

### CODE REVIEW

File: releases/2020-09/dotnet.md [hunk 1]
```diff
@@ -27,6 +27,9 @@ The Azure SDK team is pleased to announce our September 2020 client library rele
 - Key Vault (Administration, Certificates, Keys, Secrets)
 - Service Bus
 - Tables
+- Management Library - CosmosDB
+- Management Library - DNS
+- Management Library - Insight
```
Comment (nickzhums) [inline releases/2020-09/dotnet.md:33]: @allenjzhang could you take a look and see if this list is good for September

Comment (nickzhums) [reply releases/2020-09/dotnet.md:33]: confirmed with Allen this list looks good

**Comment by nickzhums** [comment]
Changed to October release notes due to SDK release plan change

**Comment by nickzhums** [comment]
@jongio those packages have all been released and we are planning to post the blog soon - need this to be approved before we can release the blog- please take a look, thanks

**Review by samvaity — APPROVED** [review]
Java release notes lgtm

**Review by xiangyan99 — APPROVED** [review]
Python one looks good to me.

File: releases/2020-10/dotnet.md [hunk 2]
```diff
@@ -59,7 +74,7 @@ $> dotnet add package Azure.Security.KeyVault.Keys --version 4.2.0-beta.2
 $> dotnet add package Azure.Security.KeyVault.Secrets --version 4.2.0-beta.2
 
 $> dotnet add package Azure.Storage.Blobs --version 12.7.0-preview.1
-$> dotnet add package Azure.Storage.Blobs.Batch --version 12.4.0-preview.1
+$> dotnet add package Azure.Storage.Blobs.Batch --version 12.4.0-preview.1df
```
Comment (kinelski) [inline releases/2020-10/dotnet.md:77]: Was this added by accident?

**Review by kinelski — APPROVED** [review]
Just one short comment. Other than that, .NET LGTM.

File: releases/2020-10/dotnet.md (hunk 2 above)
Comment (nickzhums) [reply releases/2020-10/dotnet.md]: fixed

**Pull request merged**